src/cloudant/client.py

```diff
@@ -290,6 +290,8 @@ def create_database(self, dbname, partitioned=False, **kwargs):
         except CloudantDatabaseException as ex:
             if ex.status_code == 412:
                 raise CloudantClientException(412, dbname)
+            if ex.status_code == 400:
+                raise ex
```

Comment:
Why do we wrap `new_db.create` in a try/catch here? It looks like we're suppressing 500s and 401s too.